packages/core/parcel/src/cli.js

```diff
@@ -98,7 +98,7 @@ var hmrOptions = {
   '-p, --port <port>': [
     'set the port to serve on. defaults to $PORT or 1234',
     value => parseInt(value, 10),
-    parseInt(process.env.PORT, 10) || 1234,
+    process.env.PORT != null ? parseInt(process.env.PORT, 10) : undefined,
```

Comment:
What should happen for `PORT=invalid parcel ....`? Previously, it used `NaN || 1234 === 1234`.

Reply:
I think we should just error and exit out in that case, which it probably already does, but I made it a bit more explicit